pkgs/development/compilers/ghc/head.nix

```diff
@@ -161,7 +175,11 @@ stdenv.mkDerivation rec {
   # zsh and other shells are smart about `{ghc}` but bash isn't, and doesn't
   # treat that as a unary `{x,y,z,..}` repetition.
   postInstall = ''
-    paxmark m $out/lib/${name}/bin/${if targetPlatform != hostPlatform then "ghc" else "{ghc,haddock}"}
```

Comment:
This change is motivated by the fact that we do not necessarily build `haddock` with quick flavours. As such we fall over trying to `paxmark` a file that does not exist.

Reply:
Sounds good-- probably can collapse this to just `{ghc,haddock}` if we check for existence anyway.